src/modules/showitem.js

```diff
@@ -1,4 +1,5 @@
 import { fetchList, postLikes } from './showlist.js';
+import commentCounter from './comment-counter.js';
 
 // eslint-disable-next-line consistent-return
```

Comment:
- [ ] [OPTIONAL] Please remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :+1: